activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -759,15 +759,10 @@ def reconnect
         end
 
         def configure_connection
-          if @config[:timeout] && @config[:retries]
-            raise ArgumentError, "Cannot specify both timeout and retries arguments"
-          elsif @config[:timeout]
-            @raw_connection.busy_timeout(self.class.type_cast_config_to_integer(@config[:timeout]))
-          elsif @config[:retries]
```

Comment:
Did we just throw away the `retries` config? Or am I missing something?

Reply:
My thinking is that the deprecation needs to be released in 7.2 so that it can be removed in 8.0.

Reply:
Since beta was already cut, I don't think we can backport deprecation

Reply:
Ok, I'll deprecate for 8.0 and we can remove later 👍

Reply:
@fractaledmind any update on this?

Reply:
@zzak: Added `retries` back with a deprecation warning